### ISSUE TITLE
Add popcorn calcification

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10529]
**Preferred term label:**
Popcorn calcification

**Synonyms**
-
**Definition (free text, please give PubMed ID)**
Popcorn calcifications are areas of amorphous calcifications often with rings and arcs that resemble popped corn kernels
in the metaphysis and epiphysis around the growth plate. PMID: 12432108 

Comment: Specific type of stippling
![image](https://github.com/obophenotype/human-phenotype-ontology/assets/160679899/8b3e83b4-1e5b-4fdd-9b94-e2ec9b217086)
Source: PMID: 7403509

**Parent term (use hpo.jax.org/app)**
Epiphyseal stippling HP:0010655

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Osteogenesis Imperfecta (omim166200) PMID: 18798308
Cole-Carpenter syndrome (omim112240), PMID25683117

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000871
- label: Popcorn calcification
- parents: [Epiphyseal stippling (HP:0010655)](https://hpo.jax.org/app/browse/term/HP:0010655)
- pmid: PMID:12432108
- def: Popcorn calcifications are areas of amorphous calcifications often with rings and arcs that resemble popped corn kernels in the metaphysis and epiphysis around the growth plate.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10529
- synonyms:
